pyiron_base/_tests.py

```diff
@@ -43,9 +43,9 @@ class PyironTestCase(unittest.TestCase, ABC):
     def setUp(self):
         self.addTypeEqualityFunc(np.ndarray, self._assert_equal_numpy)
 
-    def _assert_equal_numpy(self, a, b, msg=None):
+    def _assert_equal_numpy(self, a, b, msg=""):
```

Comment:
Is this necessary? I thought with the changes in line 48 now `None` is supported and maybe the better choice.